classes/Task/Restore/RestoreComplete.php

```diff
@@ -44,6 +44,8 @@ public function run(): int
     {
         $this->logger->info($this->translator->trans('Restoration process done. Congratulations! You can now reactivate your shop.'));
         $this->next = TaskName::TASK_COMPLETE;
+
+        $this->container->getFileConfigurationStorage()->cleanAllRestoreFiles();
```

Comment:
I expect a common scenario where we a restore is run after a failed update.
In this case, the update files would not be deleted, and this process would be a fail safe.

```suggestion
        $this->container->getFileConfigurationStorage()->cleanAllRestoreFiles();
        $this->container->getFileConfigurationStorage()->cleanAllUpdateFiles();
```